settings.gradle

```diff
@@ -0,0 +1,6 @@
+rootProject.name = 'totpauth-parent'
+include ':totpauth-api'
+include ':totpauth-impl'
+
+project(':totpauth-api').projectDir = "$rootDir/totpauth-api" as File
+project(':totpauth-impl').projectDir = "$rootDir/totpauth-impl" as File
```

Comment:
Perhaps I need something here which creates the `totpauth-parent/totpauth-impl/target/totpauth-impl-0.5.1-bin.zip` file?